### ISSUE TITLE
Incorrectly resolved the PTR record caused the some command (eg netstat) exception

### ISSUE THREAD

**Issue by liwanglin12** [pi-hole/pi-hole#1749]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**
_1_

---
**[BUG REPORT]:**

Pi Hole Log:
```
Oct 24 20:30:12 dnsmasq[571]: query[PTR] 7.160.168.192.in-addr.arpa from 127.0.0.1  


Oct 24 20:30:12 dnsmasq[571]: /etc/pihole/black.list 192.168.160.7 is tracking.miui.com
```

It caused:
![snipaste20171024_203709](https://user-images.githubusercontent.com/10279535/31943397-31b82da0-b8fb-11e7-8bf8-e67438f71ceb.png)

**Comment by PromoFaux** [comment]
This is a [known issue](https://www.reddit.com/r/pihole/comments/6j3az9/urls_in_my_blacklist_are_showing_up_as_my_default/)

and will be fixed in the next release: (see: #1560 )

**Comment by liwanglin12** [comment]
That's fine. thanks!

**Comment by jacobsalmela** [comment]
Resolved in v3.2 (#1560)
